pom.xml

```diff
@@ -644,37 +644,25 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>net.revelc.code</groupId>
-                <artifactId>impsort-maven-plugin</artifactId>
-                <version>1.6.2</version>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.22.8</version>
                 <configuration>
-                    <groups>*,javax.,java.</groups>
-                    <lineEnding>LF</lineEnding>
-                    <removeUnused>true</removeUnused>
-                    <staticAfter>true</staticAfter>
+                    <java>
+                        <includes>
+                            <include>**/*.java</include>
+                        </includes>
+                        <removeUnusedImports />
+                        <eclipse>
+                            <file>eclipse-codeStyle.xml</file>
+                        </eclipse>
+                        <lineEndings>UNIX</lineEndings>
```

Comment:
@rnc the [spotless docs](https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#line-endings-and-encodings-invisible-stuff) recommend we actually don't define this, the default instead using .gitattributes and the plugin will enforce whatever we set there, I avoided doing this as I didn't want to exhaustively define file extensions and line endings for them but I can look into it if we think that would be a better solution. 

The old formatter-maven-plugin would have had the same issue.